docs/reference/contributing/connectivity/CellularInterface.md

```diff
@@ -164,50 +85,72 @@ typedef enum {
 #define MDM_PIN_POLARITY            ACTIVE_HIGH_POLARITY
 
 ```
-The current implementation does not use all pins, but you must define all of them.
-
-3. **Implement `onboard_modem_api.h`** Provide an implementation of `onboard_modem_api.h`. We provide an [example implementation](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NXP/TARGET_LPC176X/TARGET_UBLOX_C027/onboard_modem_api.c).
-
-###### Providing module modem API
-
-Only valid when **Case 4** is applicable.
 
-- If the modem is already ready to use via the UART, it may be possible to use `UARTCellularInterface` directly. Just pass its constructor the necessary pin information for the module connected to your board.
+You must define all pins. Implement `onboard_modem_api.h`. The target board must provide an implementation of the [onboard_modem_API](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/cellular/onboard_modem_api.h). Please see the [example implementation ](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NXP/TARGET_LPC176X/TARGET_UBLOX_C027/onboard_modem_api.c).
 
-- If you require custom power and reset controls, create a custom class derived from `UARTCellularInterface`, which overrides the protected `modem_init()` methods.
+##### Modifying cellular targets
 
-- If using a different connection type, you must provide access to the connection by implementing the `FileHandle` API, and then you can pass your file handle for that connection to `PPPCellularInterface`. Either use it directly, or derive from it, and pass a file handle to its constructor in the same manner as `UARTCellularInterface`.
+Setup for some predefined targets is available in [CellularTargets.h](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/common/CellularTargets.h). For a new target, you need to modify `CellularTargets.h`. In `CellularTargets.h`, you need to specify a cellular device in addition to UART pins connecting the Mbed OS CPU to the modem. For example, if you used a K64F to connect with the QUECTEL BG96, you need the following changes marked in between `// !!!!` in `CellularTargets.h`:
```

Comment:
Query: Is there a reason we chose the K64F? Also, can we use a real example that's not board-specific?